fvtest/compilertest/tests/OpCodesTest.hpp

```diff
@@ -173,15 +191,150 @@ class OpCodesTest : public TestDriver
    //Unsupported OpCodes are tested in this function
    virtual void UnsupportedOpCodesTests();
 
-   uint8_t *
-   compileOpCodeMethod(int32_t opCodeArgsNum,
+   template <typename functiontype> 
+   int32_t  
+   compileOpCodeMethod(functiontype& resultpointer,
+         int32_t opCodeArgsNum,
          TR::ILOpCodes opCode,
          char * resolvedMethodName,
          TR::DataType * argTypes,
          TR::DataType returnType,
          int32_t & returnCode,
          uint16_t numArgs = 0,
-         ...);
+         ...)
+   {
+   TR_ASSERT((numArgs % 2) == 0, "Must be called with zero or an even args, numChildArgs = %d", numArgs);
+   if ((numArgs % 2) != 0)
+      {
+      fprintf(stderr, "Error: numArgs must be called with zero or an even args, numArgs is %d", numArgs);
+      exit(-1);
+      }
+
+   OpIlInjector * opCodeInjector = 0;
+   TR::ILOpCode op(opCode);
+
+   TR::TypeDictionary types;
+
+   CmpBranchOpIlInjector cmpBranchIlnjector(&types, this, opCode);
```

Comment:
I think this is a typo `cmpBranchIlnjector` -> `cmpBranchIlInjector` (note the missing "I" in front of "njector")